scripts/makeFinalPlots_UL2018.py

```diff
@@ -528,10 +528,11 @@ def makeNonNegativeHistos(hList):
         bkgList.append('QCD')
     
     col = ROOT.TColor()
-    bkgColors = {'DYmerged': (col.GetColor("#44BA68"), col.GetColor("#389956")),
-                 'TT': (col.GetColor("#F4B642"), col.GetColor("#dea63c")),
-                 'W': (col.GetColor("#41B4DB"), col.GetColor("#3ca4c8")),
-                 'singleH': (col.GetColor("#400080"), col.GetColor("#400080")),
+    bkgColors = {'DY':    (col.GetColor("#44BA68"), col.GetColor("#389956")),
```

Comment:
Anticipating on silly review comments: should we switch to the new "default" CMS color scheme?

Reply:
for reference ([source](https://indico.cern.ch/event/1372111/contributions/5768994/subcontributions/463452/attachments/2794503/4874367/20240206_CVDProjectReport_DiversityMeeting_YiChen_v3.pdf))
![image](https://github.com/LLRCMS/KLUBAnalysis/assets/72452043/6480efbb-3df2-40cf-8027-a00501107430)

Reply:
for further reference, the [paper linked in those slides](https://arxiv.org/pdf/2107.02270.pdf) includes more palettes in case we need more than 6 colors (page 10, Table 1).